specification/confidentialledger/Microsoft.ManagedCcf/transactions.tsp

```diff
@@ -31,6 +31,8 @@ model Transaction {
   status: TransactionStatus;
 }
 
+@doc("trigger testing")
+
```

Comment:
```suggestion
```

Reply:
delete before merge

Reply:
NIT: Going forward, if possible, I recommend using our sample `specification/contosowidgetmanager` to test spec changes.